docs/src/content/en/docs/frameworks/agentic-uis/copilotkit.mdx

```diff
@@ -11,139 +11,139 @@ import Image from "next/image";
 CopilotKit provides React components to quickly integrate customizable AI copilots into your application.
 Combined with Mastra, you can build sophisticated AI apps featuring bidirectional state synchronization and interactive UIs.
 
-## Create a Mastra Project
-{/*
-LLM CONTEXT:
-This Tabs component shows commands for creating a new Mastra project using different package managers.
-Each tab displays the command for that specific package manager to create a Mastra project.
-This is the first step in setting up Mastra with CopilotKit for building AI copilot applications.
-All commands create the same Mastra project but use different package manager syntax.
-*/}
+## React Frontend Setup
 
-<Tabs items={["npx", "npm", "yarn", "pnpm"]}>
-  <Tabs.Tab>
-    ```bash copy
-    npx create-mastra@latest
-    ```
-  </Tabs.Tab>
+Set up a CopilotKit component in your React frontend
+
+<Tabs items={["npm", "yarn", "pnpm"]}>
   <Tabs.Tab>
-    ```bash copy
-    npm create mastra
+    ```bash copy filename="terminal"
+    npm install @copilotkit/react-core @copilotkit/react-ui
     ```
   </Tabs.Tab>
   <Tabs.Tab>
-    ```bash copy
-    yarn create mastra
+    ```bash copy filename="terminal"
+    yarn add @copilotkit/react-core @copilotkit/react-ui
     ```
   </Tabs.Tab>
   <Tabs.Tab>
-    ```bash copy
-    pnpm create mastra
+    ```bash copy filename="terminal"
+    pnpm add @copilotkit/react-core @copilotkit/react-ui
     ```
   </Tabs.Tab>
 </Tabs>
 
+```tsx filename="components/CopilotKitComponent.tsx" showLineNumbers copy
```

Comment:
Consider adding a paragraph between the two code blocks. One reason is it looks nicer, but it's also probably helpful to explain what the code below is for.